plugins/drpc-viewercount-update/drpc-viewercount-update.js

```diff
@@ -15,13 +15,14 @@ module.exports = {
   execute(client, data) {
     const discord = Modules.getModule('discord-rpc');
     if (discord) {
-      if (discord.settings.active) {
-        discord.setCount(Number(data['live.viewers']));
-      }
+      if (discord.settings !== undefined)
```

Comment:
this should never be undefined, <.< this should check if it is null. not undefined.

Reply:
This gave underfined in the output. Error in your code? Make sure you're not just using your custom build but also pulling a fresh clone of the repo to test your stuff.